WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewHolder.kt

```diff
@@ -76,14 +80,19 @@ class BloggingPromptCardViewHolder(
 
         adapter.loadData(card.respondents)
     }
-}
 
-private fun MySiteBloggingPrompCardBinding.showCardMenu() {
-    val quickStartPopupMenu = PopupMenu(bloggingPromptCardMenu.context, bloggingPromptCardMenu)
-    quickStartPopupMenu.setOnMenuItemClickListener {
-        return@setOnMenuItemClickListener true
+    private fun showCardMenu(bloggingPromptCardView: MaterialCardView) {
+        val quickStartPopupMenu = PopupMenu(bloggingPromptCardView.context, bloggingPromptCardView)
```

Comment:
I think we need to use the menu button as an anchor, otherwise it will look like this:
[![Image from Gyazo](https://i.gyazo.com/b79323fde5fc73885f4ceedccfaab16d.png)](https://gyazo.com/b79323fde5fc73885f4ceedccfaab16d)

Reply:
Good catch, thanks!
Fixed